proc/msg.c

```diff
@@ -115,7 +118,7 @@ static void *msg_map(int dir, kmsg_t *kmsg, void *data, size_t size, process_t *
 			return NULL;
 		}
 
-		vaddr = vm_mmap(msg_common.kmap, NULL, NULL, SIZE_PAGE, PROT_READ | PROT_WRITE, OID_PHYSMEM, bpa, flags);
+		vaddr = vm_mmap(msg_common.kmap, NULL, NULL, SIZE_PAGE, PROT_READ | PROT_WRITE, (void *)-1, bpa, flags);
```

Comment:
Perhaps move `OID_PHYSMEM` to vm/map.h instead of removing it? Internal vm_mmap API hasn't changed

Reply:
But this argument isn't oid but vm_object_t

Reply:
Yes, `OID_` prefix needs to go, e.g. `VM_OBJ_PHYSMEM`?